pykern/pkcli/ci.py

```diff
@@ -5,17 +5,47 @@
 :license: http://www.apache.org/licenses/LICENSE-2.0.html
 """
 from pykern.pkdebug import pkdp, pkdlog
+from pykern import pkunit
+from pykern.pkcli import test
+from pykern import pksetup
+import re
+
+
+_FILE_TYPE = re.compile(r'.py$')
+_EXCLUDE_FILES = re.compile( f"/{test.SUITE_D}/.*{pkunit.DATA_DIR_SUFFIX}/|/{test.SUITE_D}/.*_work/|/{pksetup.PACKAGE_DATA}/|pkdebug")
```

Comment:
There's an extra space before the f, also I see double quotes used in a few places in this pr.